modular_skyrat/modules/resleeving/code/machine.dm

```diff
@@ -0,0 +1,10 @@
+GLOBAL_LIST_EMPTY(body_records)
```

Comment:
I think that was meant for another branch, wasn't it?

Reply:
oh shit lmao